### ISSUE TITLE
Ignored AxisTitles.showSideTitle value

### ISSUE THREAD

**Issue by valentinkatic** [imaNNeoFighT/fl_chart#963]
I'm migrating my app to use version 0.50.0 of fl_chart and I found out that I can no longer have visible axis titles with invisible side titles:
```
topTitles: AxisTitles(
  sideTitles: SideTitles(showTitles: false),
  axisNameWidget: Text(
    "Chart title text",
    style: chartTitleStyle,
  ),
)
```
gives me: 
![Screenshot 2022-04-01 at 15 01 59](https://user-images.githubusercontent.com/18287632/161269611-953ebdd9-120f-43d2-9a9c-4ada38169d5a.png)

but when I want to hide all titles like:
```
topTitles: AxisTitles()
```
chart is generated correctly

![Screenshot 2022-04-01 at 15 09 40](https://user-images.githubusercontent.com/18287632/161270025-e55e00c2-8271-488d-8ed7-1a4c66b82ba3.png)

Correct behavior is to have top axis title "Chart title text" visible without side titles (0, 0.5, 1, 1.5, 2, 2.5, 3).

**Comment by imaNNeoFighT** [comment]
Oh. There is a bug.
I will fix it ASAP.

**Comment by imaNNeoFighT** [comment]
Fixed in [0.50.1](https://github.com/imaNNeoFighT/fl_chart/releases/tag/0.50.1)
